README.md

```diff
@@ -3,13 +3,13 @@
 [![Build status](https://ci.appveyor.com/api/projects/status/1av8osv0099nqw8m/branch/master?svg=true)](https://ci.appveyor.com/project/trappmartin/mcmcchain-jl/branch/master)
 [![Coverage Status](https://coveralls.io/repos/github/TuringLang/MCMCChain.jl/badge.svg?branch=master)](https://coveralls.io/github/TuringLang/MCMCChain.jl?branch=master)
 
-Implementation of Julia types for summarizing MCMC simulations and utility functions for diagnostics and visualizations. 
+Implementation of Julia types for summarizing MCMC simulations and utility functions for diagnostics and visualizations.
 
 ## Example
 The following simple example illustrates how to use Chain to visually summarize a MCMC simulation:
 ```julia
 using MCMCChain
-using Plots, StatPlots
```

Comment:
At the moment you don't need to load Plots separately when loading StatPlots. That may change in the future.